cms/static/js/factories/asset_index.js

```diff
@@ -1,6 +1,6 @@
 define([
-    'jquery', 'js/collections/asset', 'js/views/assets', 'jquery.fileupload'
-], function($, AssetCollection, AssetsView) {
+    'jquery', 'js/collections/asset', 'js/views/assets', 'studio-frontend', 'jquery.fileupload'
+], function($, AssetCollection, AssetsView, Assets) {
```

Comment:
I'm not sure that we need this change, the way I've set up the template it ought to be conditionally excluding this javascript entirely.